webapp/src/components/add_issue/add_issue.jsx

```diff
@@ -143,7 +143,7 @@ export default class AddIssue extends React.PureComponent {
                                 <React.Fragment>
                                     <TextareaAutosize
                                         style={style.textareaResizeMessage}
-                                        placeholder='Enter a title'
+                                        placeholder='Enter a title *'
```

Comment:
We usally want the asterisk to be red or probably `error-text` class in this case. Also this should be a `hint` and not `placeholder` right?

Reply:
@mickmister The `TextareaAutosize` component doesn't have any `hint` prop in it. Also, the placeholder only supports `string` type, that's why we were not able to update the color.
Should we add a `label` element and add stylings to support this?

Reply:
That's an external library we're using here. If it's not trivial to change the color of the asterisk, then maybe we just don't use an asterisk here. @asaadmahmood What do you think?

Reply:
@asaadmahmood Gentle reminder for this

Reply:
@asaadmahmood Gentle reminder for this

Reply:
@ayusht2810 We can get rid of the asterik, the button is disabled, its fine that the title needs to be entered.